cv.tex

```diff
@@ -150,102 +149,70 @@ \subsection{Programming}
+\item Accomplished interviewer with 100+ combined phone screen, in-person, or virtual interviews performed
+\item Promoted to SecEng 3 (L6) in July 2021
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
 
-\runsubsection{Phabricator}
-\descript{| Open Source Contributor \& Team Leader}
+\runsubsection{Amazon.com}
+\descript{SecEng (L4-5), Security Incident Response Team}
 
-\location{Jan 2013 – May 2013 | Palo Alto, CA \& Ithaca, NY}
+\location{June 2016 - July 2018  | Seattle, WA}
 \begin{tightitemize}
-\item Phabricator is used daily by Facebook, Dropbox, Quora, Asana and more.
-\item I created the Meme generator, the entire Lipsum application, ported Tokens to different apps, fixed many bugs and more in PHP and Shell.
-\item Led a team from MIT, Cornell, IC London and UHelsinki for the project.
+\item Responder for security incidents within the retail portion of Amazon and subsidiaries
+\item Negotiating, triaging, and resolving responsible disclosure of vulnerabilities through the security@ mailbox
+\item Created requirements and performed UAT for development of custom case management solution
+\item Co-lead the on-site network security response during the 2017 \textbf{\href{https://media.wholefoodsmarket.com/whole-foods-market-payment-card-investigation-update/}{Whole Foods}} PoS security incident
+\item Promoted to SecEng 2 (L5) in October 2017
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
-%------------------------------------------------
-% Research
-%------------------------------------------------
-
-\section{Research}
-
-\runsubsection{Cornell Robot Learning Lab}
-\descript{| Head Undergrad Research}
-
-\location{Jan 2014 – Present | Ithaca, NY}
-Worked with \textbf{\href{http://www.cs.cornell.edu/~ashesh/}{Ashesh Jain}} and \textbf{\href{http://www.cs.cornell.edu/~asaxena/}{Prof Ashutosh Saxena}} to create \textbf{PlanIt}, a tool which learns from large scale user preference feedback to plan robot trajectories in human environments. Publication submitted.
-
-\sectionspace % Some whitespace after the section
-
-%------------------------------------------------
-
-\runsubsection{Cornell Phonetics Lab}
-\descript{| Head Undergraduate Researcher}
-
-\location{Mar 2012 – May 2013 | Ithaca, NY}
-Lead the development of \textbf{QuickTongue}, the first ever breakthrough tongue-controlled game with \textbf{\href{http://conf.ling.cornell.edu/~tilsen/}{Prof Sam Tilsen}} to aid in Linguistics research. Publication submitted.
-
-\sectionspace % Some whitespace after the section
-
-%------------------------------------------------
-% Awards
 %------------------------------------------------
 
-\section{Awards} 
+\runsubsection{FireEye}
```

Comment:
(JC):
- Would it be more attention-grabbing if it was “Mandiant”? It was Mandiant, right?
- “Four on, four off…” —> “Participated in a four-on-four-off shift to monitor and analyze threat activity reported by the instrusion detection system”
- “Produced and updated” —> “Produced and maintained”
- For your last bullet, can you generalize the statement a bit like “Developed automation integrated with day-to-day tooling to help with triaging alerts and scheduling shifts”?